ui/src/containers/DashboardLayout/index.tsx

```diff
@@ -14,7 +17,50 @@ interface LayoutProps {
   Header: React.FC;
 }
 
+const adminSection = (
+  <>
+    <div className={c('header-row')}>Admin</div>
+    <Link href="/organizations">
+      <div className={c('row')}>
+        <MdBusiness />
+        <span className={c('row-text')}>Organizations</span>
+      </div>
+    </Link>
+
+    <Link href="/employees">
+      <div className={c('row')}>
+        <FaUsers />
+        <span className={c('row-text')}>Employees</span>
+      </div>
+    </Link>
+  </>
+);
+
+const project = (item: any) => {
+  return (
+    <Link href="/projects" key={item.id}>
+      <div className={c('row')}>
+        <div className={c('fallback-icon')}>
+          <FallbackIcon logo={item.logo} name={item.name} />
+        </div>
+        <span className={c('row-text')}>{item.name}</span>
+      </div>
+    </Link>
+  );
+};
+
 const Index: React.FC<LayoutProps> = (p) => {
+  const user = currentUser();
+  const role = user.role?.id;
+  const projects = user.projects ? user.projects : [];
+
+  const userSection = (
+    <>
+      <div className={c('header-row')}> Projects</div>
+      {projects.map((pro) => project(pro))}
```

Comment:
We can directly pass the function here
```suggestion
      {projects.map(project)}
```